app/assets/stylesheets/components/_footer.scss

```diff
@@ -23,6 +23,16 @@ body {
     flex-direction: row;
   }
 
+  @include at-media-max('tablet') {
+    .footer__links {
+      flex-wrap: wrap;
+    }
+    a {
+      padding: 0.5rem 0;
+    }
```

Comment:
I was originally expecting we'd offset the padding from the main container to limit the visual impact. (i.e. remove `u-padding-y` from `footer__links` below, at least for mobile). I think this could be okay though, but I'd still suggest:

- Using design system unit helpers
- Updating the PR using the template with before/after screenshots and sharing with UX for review of visual differences

```suggestion
    a {
      @include u-padding-y(1);
    }
```

Reply:
I was thinking by adding padding to the 'a' tag it would make the tap target bigger.
I'll add those screenshots and look for unit helpers.

Reply:
Yes, I think the padding on the links is the correct approach to increasing tappable area. My thinking was to _remove_ the corresponding padding from the container element. The net result is that it appears visually identical, but has a larger tap area.



Before|After
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/2ed5ed8c-cf59-45af-bfc3-b32f20648e74)<br>![image](https://github.com/18F/identity-idp/assets/1779930/0bd0b16c-b59f-43de-a1e1-d4357239759c)|![image](https://github.com/18F/identity-idp/assets/1779930/1850e99c-101f-4fc4-957b-4d2bed164c21)<br>![image](https://github.com/18F/identity-idp/assets/1779930/078ad4d3-1593-4779-95ad-c77d81b0ab6e)

Reply:
Definitely! That's updated now.